examples/pytorch/infograph/README.md

```diff
@@ -0,0 +1,163 @@
+# DGL Implementation of InfoGraph
+This DGL example implements the model proposed in the paper [InfoGraph: Unsupervised and Semi-supervised Graph-Level Representation Learning via Mutual Information Maximization](https://arxiv.org/abs/1908.01000).
+
+Paper link: https://arxiv.org/abs/1908.01000
```

Comment:
You already give the paper link above and there is no need to give it a second time. Same for grand.